From 8080f3db2bf0865063018a59f155002ed7220890 Mon Sep 17 00:00:00 2001
From: Robby Cai <R63905@freescale.com>
Date: Fri, 29 Apr 2011 15:44:00 +0800
Subject: [PATCH 945/998] ENGR00142745 MX50 RD3: Fix missing pmic config in imx5_update_defconfig

Miss the PMIC config shall break the building.
This patch fixed this problem.

Signed-off-by: Robby Cai <R63905@freescale.com>
(cherry picked from commit bb9a153305fd016b8e27c7d2e329ba629d261776)
---
 arch/arm/configs/imx5_updater_defconfig |   16 ++++++++++++----
 1 files changed, 12 insertions(+), 4 deletions(-)

diff --git a/arch/arm/configs/imx5_updater_defconfig b/arch/arm/configs/imx5_updater_defconfig
index 58d4888..475578b 100644
--- a/arch/arm/configs/imx5_updater_defconfig
+++ b/arch/arm/configs/imx5_updater_defconfig
@@ -1,7 +1,7 @@
 #
 # Automatically generated make config: don't edit
 # Linux kernel version: 2.6.35.3
-# Tue Jan 11 21:48:16 2011
+# Fri Apr 29 15:38:03 2011
 #
 CONFIG_ARM=y
 CONFIG_HAVE_PWM=y
@@ -93,7 +93,6 @@ CONFIG_SIGNALFD=y
 CONFIG_TIMERFD=y
 CONFIG_EVENTFD=y
 CONFIG_SHMEM=y
-# CONFIG_ASHMEM is not set
 CONFIG_AIO=y
 CONFIG_HAVE_PERF_EVENTS=y
 CONFIG_PERF_USE_VMALLOC=y
@@ -489,6 +488,7 @@ CONFIG_MTD_CFI_I2=y
 #
 # CONFIG_MTD_DATAFLASH is not set
 CONFIG_MTD_MXC_DATAFLASH=y
+# CONFIG_MTD_MXC_M25P80 is not set
 # CONFIG_MTD_M25P80 is not set
 # CONFIG_MTD_SST25L is not set
 # CONFIG_MTD_SLRAM is not set
@@ -700,7 +700,9 @@ CONFIG_TOUCHSCREEN_MXC=y
 # CONFIG_TOUCHSCREEN_TSC2007 is not set
 # CONFIG_TOUCHSCREEN_W90X900 is not set
 # CONFIG_TOUCHSCREEN_TPS6507X is not set
+# CONFIG_TOUCHSCREEN_P1003 is not set
 # CONFIG_TOUCHSCREEN_DA9052 is not set
+# CONFIG_TOUCHSCREEN_MAX11801 is not set
 CONFIG_INPUT_MISC=y
 # CONFIG_INPUT_AD714X is not set
 # CONFIG_INPUT_ATI_REMOTE is not set
@@ -712,6 +714,8 @@ CONFIG_INPUT_MISC=y
 CONFIG_INPUT_UINPUT=y
 # CONFIG_INPUT_PCF8574 is not set
 # CONFIG_INPUT_GPIO_ROTARY_ENCODER is not set
+# CONFIG_INPUT_DA9052_ONKEY is not set
+# CONFIG_INPUT_ISL29023 is not set
 
 #
 # Hardware I/O ports
@@ -1016,6 +1020,7 @@ CONFIG_REGULATOR=y
 # CONFIG_REGULATOR_TPS65023 is not set
 # CONFIG_REGULATOR_TPS6507X is not set
 CONFIG_REGULATOR_MC13892=y
+CONFIG_REGULATOR_MC34708=y
 CONFIG_REGULATOR_LTC3589=y
 CONFIG_REGULATOR_MAX17135=y
 CONFIG_REGULATOR_DA9052=y
@@ -1216,7 +1221,7 @@ CONFIG_FB_MXC_SYNC_PANEL=y
 # CONFIG_FB_MXC_LDB is not set
 # CONFIG_FB_MXC_CLAA_WVGA_SYNC_PANEL is not set
 # CONFIG_FB_MXC_SEIKO_WVGA_SYNC_PANEL is not set
-# CONFIG_FB_MXC_SII9022 is not set
+# CONFIG_FB_MXC_SII902X is not set
 # CONFIG_FB_MXC_CH7026 is not set
 # CONFIG_FB_MXC_TVOUT_CH7024 is not set
 # CONFIG_FB_MXC_ASYNC_PANEL is not set
@@ -1468,6 +1473,7 @@ CONFIG_MXC_IPU_V3=y
 CONFIG_MXC_PMIC=y
 # CONFIG_MXC_PMIC_MC13783 is not set
 CONFIG_MXC_PMIC_MC13892=y
+CONFIG_MXC_PMIC_MC34708=y
 CONFIG_MXC_PMIC_I2C=y
 CONFIG_MXC_PMIC_SPI=y
 # CONFIG_MXC_PMIC_MC34704 is not set
@@ -1483,6 +1489,9 @@ CONFIG_MXC_MC13892_LIGHT=y
 # CONFIG_MXC_MC13892_BATTERY is not set
 CONFIG_MXC_MC13892_CONNECTIVITY=y
 CONFIG_MXC_MC13892_POWER=y
+CONFIG_MXC_MC34708_ADC=y
+CONFIG_MXC_MC34708_RTC=y
+# CONFIG_MXC_MC34708_BATTERY is not set
 # CONFIG_MXC_PMIC_MC9S08DZ60 is not set
 
 #
@@ -1511,7 +1520,6 @@ CONFIG_MXC_MC13892_POWER=y
 # MXC VPU(Video Processing Unit) support
 #
 # CONFIG_MXC_VPU is not set
-# CONFIG_MXC_VPU_IRAM is not set
 
 #
 # MXC Asynchronous Sample Rate Converter support
-- 
1.7.4.1

